src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -76,16 +76,35 @@ <h3>Using transported core dumps on Windows</h3>
  is setup:
  </p>
 
-<p>
+<code>
  set PATH=%JAVA_HOME%\bin;%JAVA_HOME%\bin\server;%PATH%
-</p>
+</code>
 
 <p>
 You can also include user JNI libraries in <b>PATH</b>.
 If the windows libraries are not identical, then they may also need to be copied
 to the debugger machine and included in <b>PATH</b>.
 </p>
 
+<p>
+By default symbols are also located using <b>PATH</b>. However, there are also Java properties that can be used to specify both the location of the binaries, and also separately the location of symbols. Use <b>sun.jvm.hotspot.debugger.windbg.imagePath</b> for the location of binaries, and use <b>sun.jvm.hotspot.debugger.windbg.symbolPath</b> for the location of symbols. <b>imagePath</b> defaults to <b>PATH</b> if not set, and <b>symbolPath</b> defaults to <b>imagePath</b>. The advantage of using these propeties is that you don't need to change your <b>PATH</b> setting, and they allow for binaries to be located separately from symbols.
+</p>
+
+<p>
+How you set these properties will depend on the SA tool being used. The following example demonstrates how to set one of the properties when launching the clhsdb tool:
+</p>
+
+<code>
+jhsdb -J-Dsun.jvm.hotspot.debugger.windbg.imagePath="%PATH%;D:/SomePath" clhsdb
```

Comment:
Forward slash does work in a dir PATH in Windows, but of course backslash is more normal.